src/main/java/arb/commons/core/Messages.java

```diff
@@ -8,6 +8,10 @@ public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_CLIENT_DISPLAYED_INDEX = "The client index provided is invalid";
+
+    public static final String MESSAGE_INVALID_PROJECT_DISPLAYED_INDEX = "The project index provided is invalid";
```

Comment:
Just checking, is this line meant to be separated from all the rest? I think they can all just be clustered together.

Reply:
Alright, I have clustered them together!